code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -1,3 +1,4 @@
+#define isbuild_step(A) (build_step == A)
```

Comment:
I'd rather you didn't user this define

Reply:
Should I move it to helpers?

Reply:
You want me to revert or move the definition?